### PR TITLE
Add initial workflow script

### PR DESCRIPTION
**Pull request by victorlin**

### CODE REVIEW

File: tools/initial-setup.sh [hunk 1]
```diff
@@ -38,6 +38,8 @@ cat > $SETUP_SCRIPT <<-EOF
 #!/bin/bash -xe
 apt-get -yqm update
 apt-get -yq upgrade python3.8
```
Comment (christiam) [inline tools/initial-setup.sh:40]: Is `python3.8` needed in addition to `python3.8-dev`?
```suggestion
```

Comment (karasikov) [reply tools/initial-setup.sh:40]: I guess you're right, it's a dependency and would be installed automatically. But it doesn't hurt to have that line explicitly, though we can remove it.

Comment (christiam) [reply tools/initial-setup.sh:40]: Thanks for the clarification. I agree it doesn't hurt to leave it there, so let's leave it as is.

Comment (victorlin) [reply tools/initial-setup.sh:40]: I had the same question here: https://github.com/NCBI-Codeathons/psss-team4/pull/3#discussion_r719040253

But yeah, we can just leave it for now.

**Pull request merged**